README.md

```diff
@@ -42,20 +42,67 @@ npm install
 
 ## Usage
 
-Example signed VC:
-
-```json
+The following code snippet provides a complete example of digitally signing
+a verifiable credential using this library:
+
+```javascript
+// create the unsigned credential
+import jsigs from 'jsonld-signatures';
+const {purposes: {AssertionProofPurpose}} = jsigs;
+import {Ed25519VerificationKey2020} from
+  '@digitalbazaar/ed25519-verification-key-2020';
+import {Ed25519Signature2020, suiteContext} from
+  '@digitalbazaar/ed25519-signature-2020';
+
+const unsignedCredential = {
```

Comment:
Move comment down to `unsignedCredential` since it seems applicable there, not on imports.

```suggestion
import jsigs from 'jsonld-signatures';
const {purposes: {AssertionProofPurpose}} = jsigs;
import {Ed25519VerificationKey2020} from
  '@digitalbazaar/ed25519-verification-key-2020';
import {Ed25519Signature2020, suiteContext} from
  '@digitalbazaar/ed25519-signature-2020';

// create the unsigned credential
const unsignedCredential = {
```